app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, in_file, out_file = command.split()
+    if command == "mv":
+        path = out_file
+        path_list = []
+        while path != "":
+            path_list.append(os.path.split(path)[1])
+            path = os.path.split(path)[0]
```

Comment:
You could obtain a path more simply without iterations or indexing, e.g.
```suggestion
        path = os.path.normpath(out_file)
        path = path.split(os.sep)
        *file_path, file_name = path
        
```
Rewrite code using this approach to avoid indexing and iterations through the path.